screens/AppNavigator/screens/BalancesScreen/SendScreen/SendScreen.tsx

```diff
@@ -0,0 +1,206 @@
+import { AddressToken } from '@defichain/whale-api-client/dist/api/address'
+import { MaterialIcons } from '@expo/vector-icons'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useState } from 'react'
+import { ScrollView, TouchableOpacity, View, StyleSheet, Modal } from 'react-native'
+import { useForm, Controller, Control } from 'react-hook-form'
+import { BarCodeScanner } from 'expo-barcode-scanner'
+import NumberFormat from 'react-number-format'
+import tailwind from 'tailwind-rn'
+import { PrimaryButton, Text, TextInput } from '../../../../../components'
+import { getTokenIcon } from '../../../../../components/icons/tokens/_index'
+import { PrimaryColor, PrimaryColorStyle } from '../../../../../constants/Theme'
+import { translate } from '../../../../../translations'
+import { BalanceParamList } from '../BalancesNavigator'
+
+interface QRScannerProps {
+  onScan: (code: string) => void
+  isQRScanning: boolean
+  onCloseModal: () => void
+}
```

Comment:
Move all QR features into another PR, don't prioritize this yet. We will track that individually.

Reply:
Ok, do you want me to remove them here? It's already working well so I'm not sure what's the point of moving it out